### DISCUSSION TITLE
Has anyone had any sucess getting Blazorise to work with Oqtane ?

### DISCUSSION THREAD

**Discussion by vnetonline** [oqtane/oqtane.framework#3248] [Q&A]
Blazorise has been gaining popularity and it would be good to get it to work with oqtane

**Comment by vnetonline** [comment]
I just had a good but it seems that Blazorise uses JavaScript modules and loads its own JavaScript file dynamics and doesn't play nice with Oqtane

**Reply by thabaum** [reply]
@vnetonline Are any of the issues related to any reported bugs for Blazorise you are experiencing?  

[https://github.com/Megabit/Blazorise/issues](https://github.com/Megabit/Blazorise/issues)

Seems like they have a lot...

Also from just taking a quick look it looks like there are 2 issues relating to JavaScript they point out on their GitHub project homepage that maybe getting resolved.

[https://github.com/Megabit/Blazorise#21-javascript-resources](https://github.com/Megabit/Blazorise#21-javascript-resources)

Not sure if this is relating or not to your issues.  

I may work with Blazorise this week since you are testing this UI Library.  I have it on my to-do list I will share any findings when I do if interested.
